clang/lib/Sema/SemaDecl.cpp

```diff
@@ -12163,6 +12163,23 @@ bool Sema::CheckFunctionDeclaration(Scope *S, FunctionDecl *NewFD,
     }
   }
 
+  // C++11 [dcl.constexpr]p1: An explicit specialization of a constexpr
```

Comment:
Update this to the latest wording please!  it is actually no longer specific to functions.